indexer-app/src/cmr/indexer/data/concepts/granule.clj

```diff
@@ -161,10 +161,14 @@
             :version-id.lowercase (when Version (s/lower-case Version))
             :version-id.lowercase-doc-values (when Version (s/lower-case Version))
 
+            :native-id native-id
+            :native-id.lowercase (s/lower-case native-id)
+            
```

Comment:
I don't understand what fields like provider-id-doc-values is for. If I add a native-id-doc-values, it won't work. It works without it.